funnel/views/venue.py

```diff
@@ -80,16 +80,18 @@ def new_venue(self):
     @requires_permission('edit-venue')
     def update_venue_settings(self):
         if request.json is None:
-            return {'error': 'Invalid data'}, 400
+            return {'error': _("Invalid data")}, 400
```

Comment:
Shouldn't this be `{'error': 'invalid_data', 'error_description': _("Invalid data")}`?